app/src/main/java/seemoo/fitbit/activities/MainActivity.java

```diff
@@ -270,4 +243,64 @@ private void scanForLastDevice(){
             Log.e(TAG, "Error: MainActivity.fitbitScan, Bluetooth not enabled");
         }
     }
+
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Checks if the user granted permission to access fine location.
+     */
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {

```

Comment:
can we not come up with a generic variant in RequestPermissionsActivity, so we do not have to copy this into every activity?

Reply:
If I remember correctly, putting it in RequestPermissionsActivity doesn't work (for whatever reason). 
I'm going to test it again during our session today.